_includes/patterns/race-and-ethnicity.html

```diff
@@ -19,11 +19,6 @@ <h5>Race</h5>
         value="black-or-african-american" />
       <label class="usa-checkbox__label" for="black-or-african-american">Black or African American</label>
     </div>
-    <div class="usa-checkbox">
-      <input class="usa-checkbox__input" id="hispanic-latinx-or-spanish" type="checkbox" name="hispanic-latinx-or-spanish"
-        value="hispanic-latinx-or-spanish" />
-      <label class="usa-checkbox__label" for="hispanic-latinx-or-spanish">Hispanic, Latinx, or Spanish</label>
-    </div>
```

Comment:
Removed so we don't combine race and ethnicity in the same question, per current OMB requirements